src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -47,9 +47,9 @@
           "description": "A unique set of strings that provide additional information for the stack.",
           "type": "array",
           "uniqueItems": true,
+          "default": [ ],
```

Comment:
This was a bug in the schema -- we had a schema whose type was `string`, but whose default value was an array. The thing whose default value is `[ ]` is not each item, but the array itself. Fixed everywhere it occurred.